aws_lambda.py

```diff
@@ -34,22 +46,35 @@ def white_bold(s):
     )
 
 
-if __name__ == "__main__":
+def run(command):
+    event = {"server": os.getenv("SERVER", "http://localhost:8888/v1")}
+    context = {"sentry_sdk": sentry_sdk}
```

Comment:
`context` used to always be `None` before. Now I turned it into a dict. The only command that knows (or cares) about this is the `uptake_health` which needs the (initialized) `sentry_sdk` so it can send events. 

Do you think this'll work for all the other commands?

Reply:
yes, context is not used anyway